.github/workflows/codespell-private.yml

```
@@ -86,5 +86,5 @@ jobs:
           persist-credentials: false
       - name: Install codespell dependencies
         run: pip install -e ".[dev]"
-      - name: Flake8 with annotations
-        uses: TrueBrain/actions-flake8@v2
+      - name: Ruff with annotations
+        run: ruff --format=github --select=ANN --ignore=ANN101,ANN401 .
```

Like this -- will it actually annotate a `flake` failure the way `actions-flake` would have? Let's find out by testing it here!

---

```suggestion
      - name: Ruff with annotations
        run: ruff --select=ANN --ignore=ANN101,ANN401 .
```
Good call.  As I say below, this private run confuses me.  This can generate errors about Python annotations but those errors do not become GitHub annotations.